ckcp/openshift_dev_setup.sh

```diff
@@ -378,6 +379,18 @@ register_compute() {
   check_cr_sync
 }
 
+install_tekton_results(){
+  APP="tekton-results"
+  echo -n "  - tekton-results application: "
+
+  kubectl apply -k "tekton-results/openshift" >/dev/null 2>&1
+
+  # Generate postgres secret
+  kubectl create secret generic tekton-results-database --namespace="openshift-pipelines" --from-literal=DATABASE_USER=postgres --from-literal=DATABASE_PASSWORD=$(openssl rand -base64 20) --dry-run=client -o yaml | kubectl apply -f - >/dev/null 2>&1
```

Comment:
I would expect `"$WORK_DIR/manifests/compute/tekton-results/tekton-results-secret.yaml"` to be created for the user at that point.
It would then be applied by `setup_compute.sh`.

Reply:
removed create secret as it shouldn't be needed.